portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,185 @@
+const submitButton = document.getElementById('submit');
+
+//create quiz questions 
+const myQuestions = [
+  {
+    question: 'Where do I go to College?',
+    answers: {
+      a: 'UCD',
+      b: 'TCD',
+      c: 'UCC'
+    },
+    correctAnswer: 'b'
+  },
+  {
+    question: 'How many dogs do I have?',
+    answers: {
+      a: 'None',
+      b: 'One',
+      c: 'Two'
+    },
+    correctAnswer: 'c'
+  },
+  {
+    question: 'Where did I intern last summer?',
+    answers: {
+      a: 'Intel',
+      b: 'Google',
+      c: 'Facebook',
+    },
+    correctAnswer: 'a'
+  },
+  {
+    question: 'What sport did I take part in in college last year?',
+    answers: {
+      a: 'Triathalon',
+      b: 'Netball',
+      c: 'Athletics',
+    },
+    correctAnswer: 'b'
+  },
+  {
+    question: 'What musical did I see recently?',
+    answers: {
+      a: 'Hamilton',
+      b: 'Hairspray',
+      c: 'Wicked',
+    },
+    correctAnswer: 'a'
+  },
+  {
+    question: 'What is my Mother\'s profession?',
+    answers: {
+      a: 'Nurse',
+      b: 'Engineer',
+      c: 'Librarian',
+    },
+    correctAnswer: 'c'
+  },
+  {
+    question: 'What concert that I was meant to go to was cancelled due to Covid-19?',
+    answers: {
+      a: 'Vampire Weekend',
+      b: 'Lewis Capaldi',
+      c: 'The Killers',
+    },
+    correctAnswer: 'c'
+  },
+  {
+    question: 'Who was my water filtration project conducted with?',
+    answers: {
+      a: 'Unicef',
+      b: 'Engineers without Boarders',
+      c: 'The Water Project',
+    },
+    correctAnswer: 'b'
+  },
+  {
+    question: 'What was the name of my mentor last summer',
+    answers: {
+      a: 'Chris',
+      b: 'Simon',
+      c: 'Cillian',
+    },
+    correctAnswer: 'a'
+  },
+  {
+    question: 'What am I studying in college?',
+    answers: {
+      a: 'Computer Science',
+      b: 'Computer Engineering',
+      c: 'Computer Systems',
+    },
+    correctAnswer: 'b'
+  }
+];
+
+function buildQuiz(){
+    const output = [];
+
+    myQuestions.forEach(
+        (currentQuestion, questionNumber)=>{
+        //store all possible answers
+        const answers = [];
+
+        for(letter in currentQuestion.answers){
+            // make radio button for each question
+            answers.push(
+                `<label>
+                  <input type="radio" name="question${questionNumber}" value="${letter}">
+                  ${letter} :
+                  ${currentQuestion.answers[letter]}
+                </label>`
+            );
+        }
+        //add question html to output array
+        output.push(
+            `<div class="question-set">
+              <div class="question"> ${currentQuestion.question} </div>
+              <div class="answers"> ${answers.join('')} </div>
+            </div>`
+        );
+    
+     }
+    );
+    //add question html to quiz html sheet
+    quizContainer.innerHTML = output.join('');
+}
+
+function showResults(){
```

Comment:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments